ui/admin/app/components/form/field/list-wrapper/select/index.hbs

```diff
@@ -0,0 +1,79 @@
+<Hds::Table class='list-wrapper-field' name={{@name}}>
+
+  <:body as |B|>
+
+    {{#each this.options as |select index|}}
+      <B.Tr>
+        <B.Td>
+          <Hds::Form::Select::Field
+            @value={{select.value}}
+            @width={{@width}}
+            disabled={{@disabled}}
+            aria-label={{t 'titles.value'}}
+            {{on 'change' (set-from-event select 'value')}}
+            as |F|
+          >
+            <F.Options>
+              {{#each @options as |selectOption|}}
+                <option
+                  value={{selectOption}}
+                  selected={{eq selectOption select.value}}
+                >
+                  {{selectOption}}
+                </option>
+              {{/each}}
+            </F.Options>
+          </Hds::Form::Select::Field>
+        </B.Td>
+
+        <B.Td>
+          {{#if this.removeOptionByIndex}}
+            <Hds::Button
+              data-test-remove-button
+              @text={{t 'actions.remove'}}
+              @color='critical'
+              @icon='trash'
+              @isIconOnly={{true}}
+              disabled={{@disabled}}
+              {{on 'click' (fn this.removeOptionByIndex index)}}
+            />
+          {{/if}}
+        </B.Td>
+      </B.Tr>
+    {{/each}}
+
+    <B.Tr>
+      <B.Td>
+        <Hds::Form::Select::Field
+          @value={{this.newOptionValue}}
+          @width={{@width}}
+          disabled={{@disabled}}
+          aria-label={{t 'titles.value'}}
+          {{on 'change' (set-from-event this 'newOptionValue')}}
+          as |F|
+        >
+          <F.Options>
+            {{#each @options as |selectOption|}}
+              <option
```

Comment:
Will the first option be selected by default? I assume it would be and there would not actually be a value selected for the select but to the user it will look like it's selected. I think we need to add a dummy value as the first option or actually have it selected

Reply:
@zed yes you are right, just tested this.  I will add a dummy option for this.

Reply:
@ZedLi added a dummy option on line 56.